internal/controllers/clustertemplate_controller.go

```diff
@@ -41,7 +41,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
 	hwv1alpha1 "github.com/openshift-kni/oran-o2ims/api/hardwaremanagement/v1alpha1"
-	oranv1alpha1 "github.com/openshift-kni/oran-o2ims/api/v1alpha1"
+	oranv1alpha1prov "github.com/openshift-kni/oran-o2ims/api/provisioning/v1alpha1"
```

Comment:
`provv1alpha1` or `provisioningv1alpha1` would be more consistent with other imports, with the version as the suffix